frontend/components/FleetAce/FleetAce.tsx

```diff
@@ -111,48 +99,44 @@ const FleetAce = ({
   return (
     <div className={wrapperClass}>
       {renderLabel()}
-      {isReady ? (
-        <AceEditor
-          ref={editorRef}
-          enableBasicAutocompletion
-          enableLiveAutocompletion
-          editorProps={{ $blockScrolling: Infinity }}
-          fontSize={fontSize}
-          mode="fleet"
-          minLines={2}
-          maxLines={20}
-          name={name}
-          onChange={onChange}
-          onLoad={fixHotkeys}
-          readOnly={readOnly}
-          setOptions={{ enableLinking: true }}
-          showGutter={showGutter}
-          showPrintMargin={false}
-          theme="fleet"
-          value={value}
-          width="100%"
-          wrapEnabled={wrapEnabled}
-          commands={[
-            {
-              name: "commandName",
-              bindKey: { win: "Ctrl-Enter", mac: "Ctrl-Enter" },
-              exec: handleSubmit,
-            },
-            {
-              name: "deleteSelection",
-              bindKey: { win: "Delete", mac: "Delete" },
-              exec: () => handleDelete("del"),
-            },
-            {
-              name: "backspaceSelection",
-              bindKey: { win: "Backspace", mac: "Backspace" },
-              exec: () => handleDelete("backspace"),
-            },
-          ]}
-        />
-      ) : (
-        <Spinner />
```

Comment:
With or without the 0ms timeout, there's not enough time for the spinner to ever display.